src/persistence.rs

```diff
@@ -113,11 +113,8 @@ fn load_regressor_without_weights(
     let mut mi = model_instance::ModelInstance::new_from_buf(input_bufreader)
         .expect("Loading model instance from regressor failed");
 
-    match cmd_arguments {
-        Some(cmd_args) => {
-            model_instance::ModelInstance::update_hyperparameters_from_cmd(cmd_args, &mut mi)?;
-        }
-        _ => (),
+    if let Some(cmd_args) = cmd_arguments {
+        model_instance::ModelInstance::update_hyperparameters_from_cmd(cmd_args, &mut mi)?;
```

Comment:
I think we discussed let Some vs. match sometime ago - not sure what was the preferred choice @bbenshalom wdyt?

Reply:
I'm fine with either - "if let" doesn't antagonize clippy so we can keep it

Reply:
This was actually a suggestion by clippy :)

Reply:
Yes, well aware :)